testsuites/CBLTester/CBL_Functional_tests/APITests/test_replication_api.py

```diff
@@ -0,0 +1,71 @@
+import pytest
```

Comment:
Is this needed? The replication functional tests cover most of the APIs

Reply:
Removed the API Tests.